README.md

```diff
@@ -128,7 +128,7 @@ The following are valid query attributes:
 | `id` | The unique ID that is associated with each Dataset |
 | `keyword` | The topic of the Dataset |
 | `modified` | The date the File was last modified |
-| `resource_uri` | Location of the File |
+| `irm_url` | Location of the IRM document |
```

Comment:
❓ Should the irm url be queryable?  Seems like we would store it but we wouldn't query allow queries by it.

Reply:
The vox populi has spoken! I'll pull the IRM URL out of the queryable attributes list.
I'll update the Solr schema accordingly.